### PR TITLE
Minor cleanup to match the 0.8 style

### PR DESCRIPTION
**Pull request by notwaldorf**
Mostly update `bower.json`, the `index` docs page, and clean up the indentation.

### CODE REVIEW

**Comment by notwaldorf** [comment]
@cdata PTAL.

File: demo/index.html [hunk 1]
```diff
@@ -1,54 +1,54 @@
...
+    <meta charset="UTF-8">
+    <title>paper-spinner demo</title>
+    <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+    <link rel="import" href="../paper-spinner.html">
+    <style>
 
-    paper-spinner.blue .circle { border-color: #4285f4; }
-    paper-spinner.blue::shadow .circle { border-color: #4285f4; }
+      paper-spinner.blue .circle { border-color: #4285f4; }
+      paper-spinner.blue::shadow .circle { border-color: #4285f4; }
```
Comment (cdata) [inline demo/index.html]: I wonder if this could be made moar awesomer with custom properties?

**Comment by cdata** [comment]
LGTM :+1:

File: demo/index.html (hunk 1 above)
Comment (notwaldorf) [inline demo/index.html]: Yeah! I will make a much-gianter PR where I'll clean-up the actual div names in the `paper-spinner` itself, and then the custom properties will make sense. Thanks for suggesting it! :raised_hands:

**Pull request merged**